### DISCUSSION TITLE
Wanting to get into laser cutting / engraving

### DISCUSSION THREAD

**Discussion by mhngithub** [arkypita/LaserGRBL#2003] [Q&A]
I need to know just what do I need to get into laser cutting and engraving.
I assume that I need the following:
1. LaserGRBL software (How much does it cost??)
2. A Laser engraving machine (EG Atomstack? I know the prices for this)

Do I need some kind of interface Between the Laser machine (EG Atomstack) and my PC?
Is a USB port all I need for the PC?

Thanks

**Comment by StuartB4** [comment]
LaserGRBL is completely free, and yes just a USB connection from the engraver, as long as it's not a apple mac computer. 
Latest release is here. Just download the install.exe file. 
https://github.com/arkypita/LaserGRBL/releases

**Reply by mhngithub** [reply]
Thanks StuartB4
I am assuming that all of the laser devices, including the Atomstack has a USB interface.
Is there an automatic setting for laser beam strength?
I am assuming that cutting 1/16" plastic needs much less energy than 1/16" steel
